### DISCUSSION TITLE
Wavelog vs Cloudlog

### DISCUSSION THREAD

**Discussion by antonov124** [wavelog/wavelog#162] [Q&A] [answered]
Where can i found the difference between Wavelog and Cloudlog? ;) 
The wiki says "A migration from Cloudlog to Wavelog is easily possible if your existing Cloudlog installation is version 2.6.3 or earlier. You don't have to do any database changes!"    Version > 2.6.3 also works? ( At the moment i run the Version 2.6.5)

thx for the response

**Comment by HB9HIL** [comment]
**Dear @antonov124**

We deliberately refrain from making a direct comparison between Wavelog and Cloudlog. Nor do we claim that one of the two versions is better or worse.

We, the core team consisting of LA8AJA, DF2ET, DJ7NT, and HB9HIL, forked from Cloudlog to Wavelog at the beginning of 2024 in order to pursue our own pace unhindered in the development of this software. Our goal is to continuously improve and further develop this logging program with a consistent speed, focusing on stability, usability, and scalability.

This evolution includes not only bug fixes and data security but also **new features**. You can find details about the new features and other innovations at Wavelog in our [Release Notes](https://github.com/wavelog/wavelog/releases). Next to a huge amount of bugfixes, new map in Logbook Advanced, new Awards, new and enhanced design elements we have improven the stability of the whole codebase. We strive to document our work as comprehensively as possible, but the comparison between Wavelog and Cloudlog, and which fits better your needs, is something you have to do and decide by your own.

So, to answer your question: we neither have nor want to make a direct comparison between the two programs. However, we can tell you that as a core team, we will no longer be working on Cloudlog as a team-unit; our focus is now on Wavelog.

Regarding the migration of versions > 2.6.3, I will further update our [wiki article about the migration](https://github.com/wavelog/Wavelog/wiki/How-to-migrate-Cloudlog-to-Wavelog) in the next few days. Just check back in a few days.

Thank you for your question. :-)

We appreciate any feedback.

73

**Comment by antonov124** [comment]
Hello HB9HIL,
ill checked the wiki again, but still the same ;)

greetings

**Comment by HB9HIL** [comment, marked as answer]
@antonov124 

Sorry was really busy the last few days:

[The wiki is updated now](https://github.com/wavelog/wavelog/wiki/How-to-migrate-Cloudlog-to-Wavelog)
